components/calendar/src/astronomy.rs

```diff
@@ -93,6 +100,25 @@ impl Location {
     pub(crate) fn local_from_universal(universal_time: Moment, location: Location) -> Moment {
         universal_time + Self::zone_from_longitude(location.longitude)
     }
+
+    /// Given a UTC-offset in hours and a Moment in universal time,
+    /// return the Moment in standard time in the time zone with the given offset.
+    /// The field utc_offset should be within the range of possible offsets given by
+    /// the constant fields `MIN_UTC_OFFSET` and `MAX_UTC_OFFSET`.
+    #[allow(dead_code)] // TODO: Remove dead_code tag after use
+    pub(crate) fn standard_from_universal(universal_moment: Moment, utc_offset: f64) -> Moment {
```

Comment:
Suggestion: it could be named `local_from_utc` for a bit more consistency with Temporal